### DISCUSSION TITLE
希望能增加同步数据到 Notion 的功能

### DISCUSSION THREAD

**Discussion by SkysCrystal** [Saszr/rfmo#5] [Ideas]
我的大部分写作素材都在 Notion ，希望能再将日常思考同步进去。而且在 Notion 中，对信息可以做更多处理。

**Comment by Saszr** [comment]
> 我的大部分写作素材都在 Notion ，希望能再将日常思考同步进去。而且在 Notion 中，对信息可以做更多处理。

优先级排后面吧
